applicationconfig/client/src/main/java/com/azure/applicationconfig/ConfigurationAsyncClient.java

```diff
@@ -322,7 +324,7 @@ private static String getETagValue(String etag) {
     }
 
     private static String getSelectQuery(SettingFields[] set) {
-        if (set == null || set.length == 0) {
+        if (ImplUtils.isNullOrEmpty(set)) {
```

Comment:
nit: I wish it was in a class called StringUtils because ImplUtils doesn't lend itself to discoverability... That's a fight for another day though. :D

Reply:
We will get there eventually. I don't want to start with a bunch of utility classes that each house a single utility :-)

Reply:
This is also check the array for being null or empty :)